From c992f460c74bee491f831a21326f51c907376ebf Mon Sep 17 00:00:00 2001
From: Leo <thinkabit.ukim@gmail.com>
Date: Thu, 10 Dec 2020 08:10:40 -0300
Subject: [PATCH] build: check for reallocarray in stdlib.h

Signed-off-by: Andrei Gherzan <andrei.gherzan@huawei.com>
Upstream-status: Backport

---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index f86cda69ac0..8fd65d13713 100644
--- a/configure.ac
+++ b/configure.ac
@@ -83,6 +83,7 @@ AC_CHECK_DECLS([
 	reallocarray],
 	[], [], [[
 #include <malloc.h>
+#include <stdlib.h>
 ]])
 
 AC_CHECK_DECLS([
