diff -up git/googletest/include/gtest/hwext/gtest-tag.h.omv~ git/googletest/include/gtest/hwext/gtest-tag.h
--- git/googletest/include/gtest/hwext/gtest-tag.h.omv~	2021-07-06 19:01:12.828013607 +0200
+++ git/googletest/include/gtest/hwext/gtest-tag.h	2021-07-06 19:01:24.291492486 +0200
@@ -3,7 +3,8 @@
 #ifndef GTEST_INCLUDE_GTEST_GTEST_TAG_H_
 #define GTEST_INCLUDE_GTEST_GTEST_TAG_H_
 
-#include <string.h>
+#include <string>
+#include <cstring>
 #include <map>
 #include <vector>
 
