From bernhard.rosenkraenzer.ext@huawei.com Thu Sep  2 10:54:01 2021
From: Bernhard Rosenkraenzer <bernhard.rosenkraenzer.ext@huawei.com>
Date: Thu,  2 Sep 2021 10:54:01 +0100
Subject: [PATCH] Fix bogus allocation

argv needs to hold 2 character pointers, not just 2 characters.

Signed-off-by: Bernhard Rosenkraenzer <bernhard.rosenkraenzer.ext@huawei.com>
Upstream-Status: Inappropriate [fixed differently upstream]

diff -up git/utils.c.omv~ git/utils.c
--- git/utils.c.omv~	2021-09-02 10:18:46.061467114 +0200
+++ git/utils.c	2021-09-02 10:21:13.011651255 +0200
@@ -268,7 +268,7 @@ close_fds(void)
 static inline void
 run_child(char *run_ptest, int fd_stdout, int fd_stderr)
 {
-	char **argv = malloc(sizeof(char) * 2);
+	char **argv = malloc(sizeof(char*) * 2);
 	chdir(dirname(strdup(run_ptest)));
 
 	argv[0] = run_ptest;
